app/models/prePopulation/PrePopulationResponse.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.prePopulation
+
+import models.BusinessDetailsResponse
+import play.api.libs.json.{Json, Writes}
+
+case class PrePopulationResponse(hasProperty: Boolean)
```

Comment:
Just noticed this needs correction it needs to return 2 flags hasUkProperty or hasNonUKProperty. Remember this is a checkbox